src/components/MovieDetails/MovieDetails.tsx

```diff
@@ -0,0 +1,45 @@
+import magnifierIcon from "../../assets/images/magnifierIcon.png";
+import { useMoviesContext } from "../../stateManagement/store";
+import Logo from "../common/Logo/Logo";
+
+import styles from "./movieDetails.module.css";
```

Comment:
![image](https://user-images.githubusercontent.com/106384184/208587249-f54592a8-b797-4b0b-882a-90cd3cce8952.png)

There must be a gap between genres